volume-group-operator/main.go

```diff
@@ -55,8 +62,20 @@ func main() {
 	opts.BindFlags(flag.CommandLine)
 	flag.Parse()
 
+	cfg := config.NewDriverConfig()
+
+	flag.StringVar(&cfg.DriverName, "driver-name", "", "The CSI driver name.")
+	flag.StringVar(&cfg.DriverEndpoint, "csi-address", "/run/csi/socket", "Address of the CSI driver socket.")
+	flag.DurationVar(&cfg.RPCTimeout, "rpc-timeout", defaultTimeout, "The timeout for RPCs to the CSI driver.")
```

Comment:
can you move this code to a util file so that we will have a function there to create all the flags that we will do in the future?

Reply:
done